internal/apis/kfd/v1alpha2/eks/create/kubernetes.go

```diff
@@ -106,11 +107,15 @@ func (k *Kubernetes) Exec() error {
 	}
 
 	if err := k.tfRunner.Init(); err != nil {
-		return fmt.Errorf("error running terraform init: %w", err)
+		logrus.Debugf("error running terraform init: %s", err)
+
+		return errPhase
```

Comment:
the error is already wrapped in the creator with a similar message: "an error occurred while running kubernetes phase", if we return errPhase here we repeat two times the same message

Reply:
the linter fails if we don't wrap the error